### ISSUE TITLE
Farming turtles can't till earth

### ISSUE THREAD

**Issue by HtoTheB** [SquidDev-CC/CC-Tweaked#1527]
### Minecraft Version

1.20.x

### Version

1.106.1

### Details

TL;DR: Farming turtles can't till earth, instead giving the message `"Nothing to dig here"`. It still worked in 1.105.

Hi there. I recently updated my CC Tweaked install from 1.105.0 to 1.106.1 and noticed that my farming turtle isn't able to till earth anymore. Instead I get the message `"Nothing to dig here"`. When installing 1.105 again and opening the exakt same world it works again.

**Steps to reproduce:** 
- Create a flatworld
- Place farming turtle above an earth block with water nearby, with an air block in between
- Open interactive shell
- `turtle.digDown()`

Placing the turtle next to the dirt block and calling `turtle.dig()` also doesn't work. 

**Link to logs:** https://gist.github.com/HtoTheB/3734706843c369b730dfbadf2c3b6c48

**Installed mods:**
	- advanced-xray-fabric 1.20-1.5.0-build.16
	- architectury 9.1.10
	- badpackets 0.4.1
	- bowinfinityfix 2.6.0
	- cloth-config 11.1.106
	- computercraft 1.106.1
	- fabric-api 0.85.0+1.20.1
	- fabricloader 0.14.21
	- indium 1.0.21+mc1.20.1
	- inventorysorter 1.9.0-1.20
	- iris 1.6.4
	- jei 14.0.0.11
	- jeresources 1.4.0.222
	- krypton 0.2.3
	- lightoverlay 8.0.0
	- lithium 0.11.2
	- minecraft 1.20.1
	- modmenu 7.1.0
	- mousetweaks 2.25
	- sodium 0.4.10+build.27
	- starlight 1.1.2+fabric.dbc156f
	- wthit 8.1.0

**Comment by HtoTheB** [comment]
I just created a clean install without any additional mods. Problem is still reproducable.

**Link to logs:** https://gist.github.com/HtoTheB/a69b8ed958116a2e8adee0ecf11b01ec

**Installed mods:**
- computercraft 1.106.1
- fabric-api 0.85.0+1.20.1
- fabricloader 0.14.21
- java 17
- minecraft 1.20.1

In both instances simply replacing the CC Tweaked version with 1.105.0 (from CurseForge) fixes the problem.
